assessmentModel/src/commonMain/kotlin/Result.kt

```diff
@@ -21,6 +21,16 @@ interface Result {
      * [Assessment] element.
      */
     val identifier: String
+
+    /**
+     * The start date timestamp for the result.
+     */
+    var startDateString: String
+
+    /**
+     * The end date timestamp for the result.
+     */
+    var endDateString: String?
```

Comment:
I decided that it made more sense to change the end date to `null` as a way of handling the case where the assessment may be stored in a partially completed state to be continued later.

Reply:
Thanks for tagging me Shannon. Following this ticket, I am assuming some work has begun on updating data schema. Can you please post near-final JSON schema in the corresponding JIRA CPCF-36. Just want to make sure we have one opportunity to see and comment (if any) on the proposed JSON schema before it is deployed. 

Thanks!

Reply:
Yes, this is just adding dates to all steps. Not full schemas.